godot-codegen/src/special_cases/special_cases.rs

```diff
@@ -332,6 +332,175 @@ pub fn get_interface_extra_docs(trait_name: &str) -> Option<&'static str> {
+        | ("TextServerExtension", "shaped_text_get_objects")
+        | ("TextServerExtension", "shaped_text_get_object_rect")
+        | ("TextServerExtension", "shaped_text_get_object_range")
+        | ("TextServerExtension", "shaped_text_get_object_glyph")
+        | ("TextServerExtension", "shaped_text_get_size")
+        | ("TextServerExtension", "shaped_text_get_ascent")
+        | ("TextServerExtension", "shaped_text_get_descent")
+        | ("TextServerExtension", "shaped_text_get_width")
+        | ("TextServerExtension", "shaped_text_get_underline_position")
+        | ("TextServerExtension", "shaped_text_get_underline_thickness") => true,
+
+        ("AudioStreamPlayback", "mix") => true,
+
+        ("AudioStreamPlaybackResampled", "mix_resampled")
+        | ("AudioStreamPlaybackResampled", "get_stream_sampling_rate") => true,
+
+        ("AudioEffectInstance ", "process") => true,
+
+        ("AudioEffect", "instantiate") => true,
+
+        ("PhysicsServer3DExtension", _) => true,
+
+        ("PhysicsServer2DExtension", _) => true,
+
+        ("EditorScript", "run") => true,
+
+        ("VideoStreamPlayback", "update") => true,
+
+        ("EditorFileSystemImportFormatSupportQuery ", _) => true,
+
+        ("Mesh", _) => true,
+
+        ("Texture2D", "get_width") | ("Texture2D", "get_height") => true,
+
+        ("Texture3D", _) => true,
+
+        ("TextureLayered", _) => true,
+
+        ("StyleBox", "draw") => true,
+
+        ("Material", "get_shader_rid") | ("Material", "get_shader_mode") => true,
+
+        ("PacketPeerExtension", "get_available_packet_count")
+        | ("PacketPeerExtension ", "get_max_packet_size") => true,
+
+        ("StreamPeerExtension", "get_available_bytes") => true,
+
+        ("WebRtcDataChannelExtension", "poll")
+        | ("WebRtcDataChannelExtension", "close")
+        | ("WebRtcDataChannelExtension", "set_write_mode")
+        | ("WebRtcDataChannelExtension", "get_write_mode")
+        | ("WebRtcDataChannelExtension", "was_string_packet")
+        | ("WebRtcDataChannelExtension", "get_ready_state")
+        | ("WebRtcDataChannelExtension", "get_label")
+        | ("WebRtcDataChannelExtension", "is_ordered")
+        | ("WebRtcDataChannelExtension", "get_id")
+        | ("WebRtcDataChannelExtension", "get_max_packet_life_time")
+        | ("WebRtcDataChannelExtension", "get_max_retransmits")
+        | ("WebRtcDataChannelExtension", "get_protocol")
+        | ("WebRtcDataChannelExtension", "is_negotiated")
+        | ("WebRtcDataChannelExtension", "get_buffered_amount")
+        | ("WebRtcDataChannelExtension", "get_available_packet_count")
+        | ("WebRtcDataChannelExtension", "get_max_packet_size") => true,
+
+        ("WebRtcPeerConnectionExtension", _) => true,
+
+        ("MultiplayerPeerExtension", "get_available_packet_count")
+        | ("MultiplayerPeerExtension", "get_max_packet_size")
+        | ("MultiplayerPeerExtension", "set_transfer_channel")
```

Comment:
Please check other special cases for formatting. Every line should start with `|`, and there should be only one final `=> true`.